lib/git.js

```diff
@@ -0,0 +1,32 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: strong-tools
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+var Promise = require('bluebird');
+var _ = require('lodash');
+var cp = require('child_process');
+var fmt = require('util').format;
+
+module.exports = git;
+
+function git(cmdAndArgs) {
```

Comment:
Wait, why do you need to create a new promise around an already resolved Promise?